universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -118,6 +119,36 @@ public void putList(Path path, ImmutableList<String> value) {
     }
   }
 
+  /**
+   * Puts the names of the repeated entities at the path. Each element in the JSON array at the path
+   * is a JSON object with an "entity_name" scalar and possibly other nested answers to questions or
```

Comment:
Suggestion (non-blocking):
```suggestion
   * is a JSON object that has at minimum a property "entity_name" that contains a string value, along with possibly other nested answers to questions or
```

Reply:
done